utils/metrics/src/main/java/org/apache/edgent/metrics/MetricsSetup.java

```diff
@@ -104,15 +104,6 @@ public MetricsSetup startJMXReporter(String jmxDomainName) {
      */
     public MetricsSetup startCSVReporter(String pathMetrics) {
 
-        if (pathMetrics == null) { // pathMetrics is NULL
-            pathMetrics = createDefaultDirectory();
-        } else {
-            File directory = new File(pathMetrics);
-            if (!directory.exists() && !directory.mkdirs()) {
-                pathMetrics = createDefaultDirectory();
-            }
-        }
-
```

Comment:
No, now it's just the argument of this method:

`public MetricsSetup startCSVReporter(String pathMetrics) {

        final CsvReporter reporter = CsvReporter.forRegistry(registry()).formatFor(Locale.US)
                .convertRatesTo(TimeUnit.SECONDS).convertDurationsTo(TimeUnit.MILLISECONDS)
                .build(new File(pathMetrics));
        reporter.start(1, TimeUnit.SECONDS);

        return this;
    }`